clustered/common/src/main/java/org/ehcache/clustered/common/internal/exceptions/DestroyInProgressException.java

```diff
@@ -26,4 +26,9 @@
   public DestroyInProgressException(String message) {
     super(message);
   }
+
+  @Override
+  public DestroyInProgressException withClientStackTrace() {
+    return new DestroyInProgressException(this.getMessage());
```

Comment:
Why was this needed?

Reply:
This I have done for scenarios in which DestroyInProgressException is caught in the client side and take action accordingly like in retrieveEntity(). But currently this exception can never be caught in client side because the Ehcache codec always throws Lifecycle Exception for that.